src/server/game/AuctionHouseBot/AuctionHouseBot.h

```diff
@@ -224,13 +225,17 @@ class TC_GAME_API AuctionBotConfig
 
     uint32 GetItemPerCycleBoost() const { return _itemsPerCycleBoost; }
     uint32 GetItemPerCycleNormal() const { return _itemsPerCycleNormal; }
+    uint32 GetRandChar() const;
+    uint32 GetRandCharExclude(uint32 exclude) const;
+    bool GetIsBotChar(uint32 characterID) const;
```

Comment:
Wasn't sure about that one, figured I'd stick with the Get* prefix the other functions have. Will change momentarily.